src/main/java/Database/songLibrary.java

```diff
@@ -9,9 +9,10 @@
  */
 public class songLibrary implements songAccessInterface{
 
+    // TODO: change "" param to filepath
     private static final songLibrary SONG_LIBRARY = new songLibrary("");
     private final HashMap<Integer, songDsData> library;
-    private String filepath;
+    private final String filepath;
```

Comment:
Whoops

Reply:
Lol, I was just clicking on all the random warnings when I pulled.